apps/portal/pages/auth/index.tsx

```
@@ -34,28 +36,32 @@ interface FlowLink {
   const router = useRouter();
   const [params, setParams] = useState('');
+  const { features, newFeatures } = useFeatures();
   const currentPathname = router.pathname.match(panelRoutes.login);
+  const { t, i18n } = useTranslation();
+  const flowLinks: FlowLink[] = [
+    {
+      name: 'login',
+      title:
+        t('page.auth.common.existingAccount') +
+        ' ' +
+        t('page.auth.common.login'),
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)